modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -87,29 +87,24 @@ class Instrument_Manager extends \NDB_Menu_Filter
                 array('testname' => $_FILES['install_file']['name'])
             );
             if ($exists <= 0) {
-                $db_config = $config->getSetting('database');
-                exec("php generate_tables_sql_and_testNames.php < $new_file");
-
-                $instrument = \NDB_BVL_Instrument::factory(
-                    $instname,
-                    '',
-                    ''
+                $error_message = exec(
+                    'php generate_tables_sql_and_testNames.php < '
+                    . escapeshellarg($new_file)
                 );
-                if (!isset($db_config['quatUser'], $db_config['quatPassword'])) {
+                if (!empty($error_message)
+                    && strpos($error_message, "Instrument") !== false
+                ) {
+                    // Instrument(s)
                     throw new \LorisException(
-                        "SQL needs to be sourced manually at /project/tables_sql/".
-                        $instrument->table.".sql"
+                        "Error uploading the Instrument(s) file:\n".
+                        "$error_message"
+                    );
+                    // See the /project/tables_sql/{file} for debugging the error.
+                    error_log(
```

Comment:
This error_log may not display if the above exception isn't handled

Reply:
See above.